genie/example_filetype_format.py

```diff
@@ -6,6 +6,46 @@
 logger = logging.getLogger(__name__)
 
 
+class ValidationResults:
+    """Validation results"""
+
+    def __init__(self, errors: str, warnings: str, detailed: str = None) -> None:
+        """
+        Args:
+            errors (str): errors for the file
+            warnings (str): warning for the file
+            detailed_errors (pd.DataFrame, optional): Dataframe of detailed row based
+                                                      error messages. Defaults to None.
+        """
+        self.errors = errors
+        self.warnings = warnings
+        self.detailed = detailed
+        self.is_valid = errors == ""
```

Comment:
Would it be safer to have `is_valid` as a function that performs an operation on `self.errors` to determine the validity?  In case the `errors` is updated in the object

Reply:
Yes, that is a really good suggestion. Will make changes